src/canvas/Renderer.tsx

```diff
@@ -15,7 +17,7 @@ class Renderer implements IRenderer {
     private _toolType: ToolType;
     private _pathSelection: PathSelection;
 
-    constructor () {
+    constructor (DocumentModel: Document, PathSelectionModel: PathSelection) {
```

Comment:
I'd keep using the convention of lowercase first character for variables and uppercase first char for classes. also, why add 'model' at the end?

Reply:
ok I understand better the model thing. I wasnt familiar with that. the comment about the lowercase start would still hold though.

Reply:
I changed the params in the constructor to camel case 👍